GameServerLib/Items/Shop.cs

```diff
@@ -47,6 +48,7 @@ public bool ItemBuyRequest(int itemId)
                 {
                     stats.RemoveModifier(item.ItemType);
                     _game.PacketNotifier.NotifyRemoveItem(_owner, inventory.GetItemSlot(item), 0);
+                    ((IChampion)_owner).RemoveSpell((byte)(inventory.GetItemSlot(item) + 6));
```

Comment:
Instead of using the `6` as a magic number, we should have a constant somewhere that specifies where item spell slots start and use that here. Similar constant already exists for runes if I'm not mistaken.

Reply:
@FurkanS1821 you should create an issue for that, and also mention the use of constants 😉

Reply:
yeah ur right